src/data/local/github/tracks/track-4/steps/step-2.md

```diff
@@ -10,14 +10,14 @@ GraphQL can be a little scary at the first sight, but don't worry, you'll get us
 - [GraphQL official page](https://graphql.org/learn)
 - [How to GraphQL](https://www.howtographql.com/). We recommend, for now, watch only the first 4 videos of this tutorial.
 
-
 ## Practice
-As we do with REST, let's make a GraphQL request using a tool called **GraphiQL** (no code yet, hang in there). You don't need to download it, just access https://tq-template-server-sample.herokuapp.com/graphiql and you're good to go.
+As we do with REST, let's make a GraphQL request using a tool called **GraphiQL** (no code yet, hang in there). You can download the Graphiql app if you want, but it's not necessary, just access https://tq-template-server-sample.herokuapp.com/graphql and you're good to go.
```

Comment:
O problema é que o GraphiQL do server não dá pra colocar headers (ou dá? 🤔)

Reply:
Dá sim, é a mesma coisa. Só que se for pelo navegador ele tem que usar uma extensão.

Reply:
@alanraso isso, por isso tinha colocado o GraphiQL que não é do server kkk para não ter que falar sobre a extensão e tal 😅

Reply:
Melhor sugerir o download mesmo